src/vaibhav/assignment36/UnionArray.java

```diff
@@ -0,0 +1,60 @@
+package vaibhav.assignment36;
+
+/*Assignment : 36 [6th Dec'2020] 
+
+Program 1: Find union of two arrays.
+Input: arr1 -> {10.45, 14.0, 18.35, 88.88, 54.10, 18.35}
+          arr2 -> {17.20, 13.30, 10.45, 18.35, 84.33, 13.30}
+
+Output: 10.45, 14.0, 18.35, 88.88, 54.10, 17.20, 13.30, 84.33
+Method Signature : double[] getUnionOfArray(double[] arr1, double[] arr2)*/
+
+public class UnionArray {
+	
+	public static double[] getUnionOfArray(double[] arr1, double[] arr2) {
```

Comment:
This code bit complex in execution, i have sent you simple on solution on skype, check that
take of this next time